### PR TITLE
Update dataset card creation

### PR DESCRIPTION
**Pull request by stevhliu**
Encourages users to create a dataset card on the Hub directly with the new metadata ui + import dataset card template instead of telling users to manually create and upload one.

### CODE REVIEW

File: docs/source/dataset_card.mdx [hunk 1]
```diff
@@ -4,27 +4,25 @@ Each dataset should have a dataset card to promote responsible usage and inform
...
+1. Go to your dataset repository on the [Hub](https://hf.co/new-dataset) and click on **Create Dataset Card** to create a new `README.md` file in your repository.
 
-2. Generate structured tags to help users discover your dataset on the Hub. Create the tags with the [online Datasets Tagging app](https://huggingface.co/spaces/huggingface/datasets-tagging).
+2. Use the **Metadata UI** to select the tags that describe your dataset. You can add a license, language, pretty_name, the task_categories, size_categories, and any other tags that you think are relevant. These tags help users discover and find your dataset on the Hub.
 
-3. Select the appropriate tags for your dataset from the dropdown menus.
+  <Tip>
 
-4. Copy the YAML tags under **Finalized tag set** and paste the tags at the top of your `README.md` file.
+  For a more complete, but not required, set of tag options you can also use the [Dataset Tagger](https://huggingface.co/spaces/huggingface/datasets-tagging). This'll have a few more tag options like `multilinguality` and `language_creators` which are useful but not absolutely necessary.
```
Comment (lhoestq) [inline docs/source/dataset_card.mdx:15]: We don't maintain the tagging app anymore, we can redirect to the specs instead:

```suggestion
  For a more complete, but not required, set of tag options you can also look at the [Dataset Card specifications](https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1). This'll have a few more tag options like `multilinguality` and `language_creators` which are useful but not absolutely necessary.
```

however this feels a bit redundant with the `More YAML tags` section, maybe we can remove it

File: docs/source/upload_dataset.mdx [hunk 2]
```diff
@@ -55,17 +55,15 @@ Adding a Dataset card is super valuable for helping users find your dataset and
     <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/dataset_card.png"/>
 </div>
 
-2. Feel free to copy this Dataset card [template](https://raw.githubusercontent.com/huggingface/datasets/main/templates/README.md) to help you fill out all the relevant fields. 
+2. At the top, you'll see the **Metadata UI** with several fields to select from like license, language, and task categories. These are the most important tags for helping users discover your dataset on the Hub. When you select an option from each field, they'll be automatically added to the top of the dataset card.
 
-3. The Dataset card uses structured tags to help users discover your dataset on the Hub. Use the [Dataset Tagger](https://huggingface.co/spaces/huggingface/datasets-tagging) to help you generate the appropriate tags.
-
-4. Copy the generated tags, paste them at the top of your Dataset card, and then commit your changes.
+    You can also use the [Dataset Tagger](https://huggingface.co/spaces/huggingface/datasets-tagging), which has more comprehensive (but not required) tag options like `annotations_creators`, to help you generate the appropriate tags.
```
Comment (lhoestq) [inline docs/source/upload_dataset.mdx:60]:
```suggestion
    You can also look at the [Dataset Card specifications](https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1), which has more comprehensive (but not required) tag options like `annotations_creators`, to help you generate the appropriate tags.
```

**Review by lhoestq — APPROVED** [review]
Thanks ! Just two comments:

File: docs/source/dataset_card.mdx (hunk 1 above)
Comment (stevhliu) [reply docs/source/dataset_card.mdx:15]: +1 to redundancy! I removed the whole `More YAML tags` section and the link to the Hub docs, which are essentially the same.

**Comment by lhoestq** [comment]
The CI failure is unrelated to your PR - feel free to merge :)

**Comment by stevhliu** [comment]
Haha thanks, you read my mind :)

**Pull request merged**